modules/exploits/multi/http/jenkins_xstream_deserialize.rb

```diff
@@ -57,6 +61,8 @@ def initialize(info = {})
 
     register_options([
       OptString.new('TARGETURI', [true, 'The base path to Jenkins', '/']),
+      OptString.new('PSH_PATH', [false, 'Path to powershell.exe', '']),
+      OptInt.new("ListenerTimeout", [true, "Number of seconds to wait for connect back", 30]),
```

Comment:
Did you try setting `WfsDelay`? This is the standard way to increase listener timeout.